### PR TITLE
Analytics trial landing page sept 2018

### PR DESCRIPTION
**Pull request by Chandu199**
Added marketo form to the page. Completed Styling

### CODE REVIEW

File: source/analytics/index.html.erb [hunk 1]
```diff
@@ -119,27 +104,43 @@ show_features: true
...
+          <div class="form-loading-error"></div>
+          <script defer="true" src="//app-e.marketo.com/js/forms2/js/forms2.min.js"></script>
+          <script defer="true" src="/assets/javascripts/marketo_popover.js" onload= "new MarketoForm('//app-e.marketo.com', '199-QDE-291').init(9095)"></script>
+          <noscript>
+            <div class="error-msg-container">
+              <p class="error-msg_text">
+                Our signup form is blocked by your browser. You can email us directly at support@thoughtworks.com.
+              </p>
+            </div>
+            <style>
```
Comment (arvindsv) [inline source/analytics/index.html.erb]: Why is there an inline style here?

File: source/assets/stylesheets/scss/_campaign.scss [hunk 2]
```diff
@@ -1336,4 +1310,185 @@ h1.main-caption {
...
+            }
+        }
+
+        .testimonial-photo {
+            width: 150px;
+            height: 150px;
+            display: block;
+            border-radius: 50%;
+            margin: 0 1em 0 1em;
+            background: url("../../assets/images/campaign/analytics/analytics-bkgd.jpg");            background-size: cover;
```
Comment (arvindsv) [inline source/assets/stylesheets/scss/_campaign.scss]: Try using `/assets` instead.

File: source/analytics/index.html.erb [hunk 3]
```diff
@@ -16,65 +16,38 @@ show_features: true
...
-
-
-          <div class="col-md-4 feature">
-            <h3>Identify Bottlenecks<br/>in Your Process</h3>
-              <%= image_tag "campaign/analytics/bottleneck.svg", {:title => "Identify Bottlenecks in Your Process", :alt => "CD illustration", :class => 'illo'} %>
-             <p>GoCD provides both in-context pipeline analytics and top-level metrics to illuminate bottlenecks in your deployment process and help you streamline it.</p>
+      <div class="row cta-row">
+        <% link_to '/enterprise/contact/index.html' do %>
+          <div class="cta-button">
+            <h3>START FREE TRIAL</h3>
```
Comment (arvindsv) [inline source/analytics/index.html.erb]: Ideally, upper-casing should be handled through CSS.

File: source/analytics/index.html.erb (hunk 1 above)
Comment (Chandu199) [reply source/analytics/index.html.erb]: Noscript tag doesnt take style written in Stylesheet. It needs Inline style

**Comment by maheshp** [comment]
Is the PR rebased, why do I see commits from master.

**Comment by maheshp** [comment]
@Chandu199 also do you want to consider squashing your commits.

File: source/analytics/index.html.erb (hunk 1 above)
Comment (Chandu199) [reply source/analytics/index.html.erb]: @arvindsv  The style for footer I have written should only take effect when the JS is turned off thats that reason it is inline style  inside noscript.

Comment (arvindsv) [reply source/analytics/index.html.erb]: That doesn't explain the code I pasted above. But, it's fine.

**Comment by arvindsv** [comment]
Squash merged here: https://github.com/gocd/www.go.cd/commit/ef48e088652aff61bd6700498fd17869d15005e5

**Comment by arvindsv** [comment]
@Chandu199 I had to fix smooth scrolling. It was not working and @Huimin wanted me to take a look. See https://github.com/gocd/www.go.cd/pull/825/commits/3b8fe0711544bce08821ec8368446265cff1fab1 - and if there's anything wrong with it, feel free to add a new commit and open a PR.

**Pull request closed without merging**